samples/open-telemetry/logging/Core_8/GenericHost/GenericHost.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.EventLog" Version="6.*" />
     <PackageReference Include="NServiceBus" Version="8.*" />
     <PackageReference Include="NServiceBus.Extensions.Hosting" Version="2.*" />
-    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc9.4" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.6" />
```

Comment:
Consider where this should be 1.* instead of 1.6

Reply:
thanks for the feedback, I've updated the version